### PR TITLE
Adds GitHub Issue Analysis Scripts

### PR DESCRIPTION
**Pull request by akestner**
**1. Issue, if available:**
N/A

**2. Description of changes:**
This PR adds a couple simple python scripts to analyze the GitHub issues for the project. The first, `hack/feature_request_reactions.py`, creates a CSV file with the number of "+1s" for each feature request. The second, `hack/label_issue_count.py`, counts the number of open issues for each label. I've also included a Makefile target (`make issues`) for easy consumption.

**3. Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [X] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: hack/label_issue_count.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if label.name not in issue_label_counts.keys():
+      issue_label_counts[label.name] = 1
+    else:
+      issue_label_counts[label.name] += 1
+
+label_row_list = [['Label', 'Issue Count']]
+for label in sorted(issue_label_counts, key=issue_label_counts.get, reverse=True):
+  label_row_list.append([label, issue_label_counts[label]])
+
+with open(os.path.expanduser('karpenter-labels-%s.csv' % (date.today())), 'w') as file:
```
Comment (ellistarn) [inline hack/label_issue_count.py:37]: Thoughts on writing to stdout to avoid leaking? I'm always wary of keeping a gitignore up to date.

**Review by ellistarn — DISMISSED** [review]
Nice! Optional comment

File: hack/label_issue_count.py (hunk 1 above)
Comment (akestner) [reply hack/label_issue_count.py]: Updated, put a `*.csv` in `.gitignore` to prevent accidental commits of the output

**Review by ellistarn — APPROVED** [review]
Nice!

**Pull request merged**